### DISCUSSION TITLE
Audiences wont work with "X-Content-Type-Options":"nosniff" - Mime Type

### DISCUSSION THREAD

**Discussion by bde-maze** [vercel/vercel#8829] [Help] [answered]
Hello there,

I'm trying out Audiences and it does't seems to work when the security headers are set in this way:
```js
{
    key: "X-Content-Type-Options",
    value: "nosniff",
}
```

I get an error in the console:
"Refused to execute script from 'https://DOMAIN.COM/va/script.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled."

What would be the correct way to set the headers while keeping a safe setting ?

Thank you

**Comment by amyegan** [comment, marked as answer]
Hi @bde-maze! We have an existing conversation about this over in [Community Discussions](https://github.com/vercel/community/discussions/1000) and an [issue in the analytics repo](https://github.com/vercel/analytics/issues/1). Come join us there to keep the conversation all in one place! https://github.com/vercel/community/discussions/1000
